### PR TITLE
Updating README to include language to recommend review/rebuild before use

### PR DESCRIPTION
**Pull request by samcofer**
The security team receives a number of tickets and escalations about the CVE/Vulnerability results of our image scans. We can't control certain elements of our images and so we're not easily going to be able to keep our images CVE free, as much as we'd like too. But what we should do is include language in the README that specifically calls the security risks of our existing images and that we recommend customers review and rebuild the images prior to production use.

### CODE REVIEW

**Comment by samcofer** [comment]
@phalston Can you review this language?

**Review by ianpittwood — APPROVED** [review]
This looks good to me. Rebuilding and updating previous (but still supported) versions is something we want to get to, see #498.

File: content/README.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ these pro images. Attempting to build a pro image using a content-base image tha
 does not exist will result in an error.
 
 
+
```
[the remark this answers is not in the document]
Comment (samcofer) [reply content/README.md]: Yup, it's been removed! Thanks for catching it!

**Review by tdstein — APPROVED** [review]

**Review by colearendt — APPROVED** [review]
LGTM! My only potential addition would be to add something to the effect of "reach out to your customer representative if you have any questions." Not essential to have wording like that though 😄

**Comment by samcofer** [comment]
Thanks @colearendt, I thought about that, but CS is considering having customers with no CS, so we're trying to avoid routing language in new communications that redirect to that routing.

**Pull request merged**